src/Module.php

```diff
@@ -136,31 +151,31 @@ public function getViewHelperConfig()
     }
 
     /**
-     * @inheritdoc
+     * @inheritDoc
      */
     public function getServiceConfig()
     {
         return [
-            'aliases' => [
-                'Laminas\DeveloperTools\ReportInterface' => Report::class,
+            'aliases'    => [
+                ReportInterface::class => Report::class,
 
                 // Legacy Zend Framework aliases
-                'ZendDeveloperTools\ReportInterface' => 'Laminas\DeveloperTools\ReportInterface',
-                \ZendDeveloperTools\Report::class => Report::class,
+                'ZendDeveloperTools\ReportInterface'    => ReportInterface::class,
+                \ZendDeveloperTools\Report::class       => Report::class,
                 'ZendDeveloperTools\ExceptionCollector' => 'Laminas\DeveloperTools\ExceptionCollector',
-                'ZendDeveloperTools\RequestCollector' => 'Laminas\DeveloperTools\RequestCollector',
-                'ZendDeveloperTools\ConfigCollector' => 'Laminas\DeveloperTools\ConfigCollector',
-                'ZendDeveloperTools\MailCollector' => 'Laminas\DeveloperTools\MailCollector',
-                'ZendDeveloperTools\MemoryCollector' => 'Laminas\DeveloperTools\MemoryCollector',
-                'ZendDeveloperTools\TimeCollector' => 'Laminas\DeveloperTools\TimeCollector',
-                'ZendDeveloperTools\FlushListener' => 'Laminas\DeveloperTools\FlushListener',
-                \ZendDeveloperTools\Profiler::class => Profiler::class,
-                'ZendDeveloperTools\Config' => 'Laminas\DeveloperTools\Config',
-                'ZendDeveloperTools\Event' => 'Laminas\DeveloperTools\Event',
-                'ZendDeveloperTools\StorageListener' => 'Laminas\DeveloperTools\StorageListener',
```

Comment:
All of these aliases are better off being `'Zend\FQCN' => Laminas\FQCN::class` so that psalm doesn't complain about unknown classes - should reduce the baseline a bit

Reply:
Agreed, will sort